WooCommerce/WooCommerceTests/ViewRelated/Survey/SurveyViewControllerTests.swift

```diff
@@ -22,21 +22,6 @@ final class SurveyViewControllerTests: XCTestCase {
         XCTAssertEqual(mirror.webView.url, WooConstants.URLs.inAppFeedback.asURL().tagPlatform("ios").tagAppVersion(Bundle.main.bundleVersion()))
     }
 
-    func test_it_loads_the_correct_product_feedback_survey() throws {
-        // Given
-        let viewController = SurveyViewController(survey: .productsVariationsFeedback, onCompletion: {})
```

Comment:
What do you think of keeping this unit test, but testing the `productsFeedback` survey case?

Reply:
That makes sense, thanks for your attention to detail! :D